test/utility.cc

```diff
@@ -16,6 +16,8 @@
 
 namespace proxy_wasm {
 
+std::string TestContext::global_log_;
```

Comment:
Does this need to be declared both here and in `TestContext`?

Reply:
Because it is `static`, it needs to be declared in `utility.h` and defined in `utility.cc`.